### PR TITLE
creating a landing page for on campus campaign

### PR DESCRIPTION
**Pull request by philspray**
### Trello card

https://trello.com/c/N89i7gek

### Context

The on campus team are running a promotional campaign for final year students

### Changes proposed in this pull request

Creating a landing page to support the campaign

### CODE REVIEW

File: app/views/content/landing/inspire-the-next-generation/_content.html.erb [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<div class="row">
+  <section class="col col-720">
+    <h2 class="heading--box-blue">How much could you earn as a teacher?</h2>
```
Comment (EllieNodder) [inline app/views/content/landing/inspire-the-next-generation/_content.html.erb:3]: Should these headers match the headings at the top of the page?

File: app/views/content/landing/inspire-the-next-generation/_content.html.erb [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+    <div class="inset">
+      <p>
+        Postgraduate teacher training course fees are around £9,250 per year.
+        You can apply for tuition fee and maintenance loans, even if you already have a student loan.
+      </p>
+
+      <p>
+        If you're interested in teaching certain subjects, you might be able to get a scholarship or bursary of up to £29k tax-free.
+        This is to support you while you are training and does not need to be paid back.
```
Comment (EllieNodder) [inline app/views/content/landing/inspire-the-next-generation/_content.html.erb:30]:
```suggestion
        This is to support you while you're training and does not need to be paid back.
```

File: app/views/content/landing/inspire-the-next-generation/_content.html.erb [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+      <p>
+        Full-time postgraduate teacher training courses usually take 9 months,
+        and undergraduate courses can take up to 4 years.
+      </p>
+      <p>
+      Most courses start in September. You can start applying now for a place after you graduate.
+      </p>
+
+      <p>
+        <%= link_to("Find out how to train to teach", page_path("train-to-be-a-teacher")) %>.
```
Comment (EllieNodder) [inline app/views/content/landing/inspire-the-next-generation/_content.html.erb:51]: Feels a bit strange to be talking about applying in the header, but then we link them somewhere that isn't really about applying. If we don't want to link straight to Apply and want to keep them in GIT, could it be better to send them to our how to apply page instead?

**Comment by gemmadallmandfe** [comment]
Looks good to me @philspray - happy with this if the on campus team are

My only question is whether we should change the URL - `inspire-the-next-generation` as a URL might be something we want to save for bigger and better landing pages!

cc @EllieNodder

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**